.github/workflows/container-release.yml

```diff
@@ -0,0 +1,41 @@
+name: container-release
```

Comment:
In one of these release scripts, should we also run the release script that populates the database?

Also should this replace the other release flow?

Reply:
Yeah I think that's a good idea. Just get it all goin in one place.